.github/workflows/chromatic.yml

```
@@ -0,0 +1,47 @@
+name: Chromatic
+
+on:
+  workflow_dispatch:
+    inputs:
+      number:
+        description: "Pull Request Number"
+        required: true
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    outputs:
+      branch_name: ${{ fromJson(steps.get_pull.outputs.data).head.label }}
+    steps:
+      - id: get_pull
+        uses: octokit/request-action@v2.x
+        with:
+          route: GET /repos/{owner}/{repo}/pulls/{pull_number}
+          owner: ${{ github.event.repository.owner.login }}
+          repo: ${{ github.event.repository.name }}
+          pull_number: ${{ github.event.inputs.number }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - id: escape
```

Chromatic has `needs: check` but the job is called `escape`. 

```suggestion
      - id: check
```